code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -21,12 +21,18 @@
 		return
 	attack(mob/M as mob, mob/user as mob, def_zone)
 		if(M == user)
-			M << "\blue You [apply_method] [src]."
+			if(istype(M,/mob/living/carbon/human))
+				var/mob/living/carbon/human/H = M
+				if(H.species.flags & IS_SYNTHETIC)
+					H << "<span class='warning'>You have a monitor for a head, where do you think you're going to put that?</span>"
+					return
+
+			M << "<span class='notify'>You [apply_method] [src].</span>"
 			M.unEquip(src) //icon update
 			if(reagents.total_volume)
 				reagents.reaction(M, apply_type)
-				spawn(5)
-					reagents.trans_to(M, reagents.total_volume)
+				spawn(0)
+					reagents.trans_to_ingest(M, reagents.total_volume)

```

Comment:
> Dunno why this was changed, to be honest

Because when Goofball was tromping through the co--

> it introduces an additional delay to patches and pills that doesn't need to be there.

Oh, you mean adding back in the delay that's supposed to be there, because it was always there, until Goofball yanked *one* of the *two* delays on pills for no apparent reason. Yeah, it's because pretty much anything you eat takes time to properly enter your system, as opposed to directly injecting it.